### PR TITLE
Link to actioncable in rails/rails

### PR DESCRIPTION
**Pull request by claudiob**
Together with this PR, I suggest to remove the "Issues" tab in this project (in GitHub settings)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,485 +1,7 @@
...
-
-* http://www.opensource.org/licenses/MIT
-
-
-## Support
-
-Bug reports can be filed for the alpha development project here:
-
-* https://github.com/rails/actioncable/issues
+An archive of the source code before the merge is available [here](../../tree/archive).
```
Comment (claudiob) [inline README.md]: oops… I need to fix this link, one second

Comment (claudiob) [inline README.md]: Actually the link is correct… I was just mimicking https://github.com/rails/activejob/pull/109

Just like @chancancode suggested (and @rafaelfranca did) in that situation, **You should branch master into a new branch called "archive" before doing this**, and should probably disable the issue tracker as well.

**Comment by davidkuhta** [comment]
@claudiob 
Should discussion on  [AC #114](https://github.com/rails/actioncable/issues/114), **Add ActionCable as a topic to the guide**, switch back over to  [Rails #22178](https://github.com/rails/rails/issues/22178)?

**Comment by claudiob** [comment]
Sure, all the open pull requests that are still meaningful should be moved over (and closed here).

**Comment by rafaelfranca** [comment]
I'll move the issue back

**Pull request merged**